scripts/automation/create_release_tag.py

```diff
@@ -0,0 +1,446 @@
+}
+
+
+def get_file_diff() -> List[str]:
+    """Get added lines from git diff"""
+    diff_output = subprocess.check_output(
+        ["git", "diff", "HEAD^", "HEAD", "--", TARGET_FILE],
+        text=True
+    )
+
+    added_lines = [
+        line[1:].strip()
+        for line in diff_output.splitlines()
+        if line.startswith("+") and not line.startswith("+++")
+    ]
+
+    return added_lines
+
+
+def parse_filenames(added_lines: List[str]) -> List[str]:
+    """Parse filenames from added lines"""
+    filenames = []
+    for line in added_lines:
+        if '"filename":' in line:
+            try:
+                filename = line.split(":")[1].strip().strip('",')
+                filenames.append(filename)
+            except IndexError:
+                print(f"Error parsing line: {line}")
+    return filenames
+
+
+def parse_sha256_digest(added_lines: List[str]) -> str:
+    shas = []
+    for line in added_lines:
+        if "sha256Digest" in line:
+            try:
+                sha_match = re.search(r'"sha256Digest"\s*:\s*"([a-fA-F0-9]{64})"', line)
+                if sha_match:
+                    sha = sha_match.group(1)
+                    print(f"get sha256Digest change: {sha}")
+                    shas.append(sha)
+            except IndexError:
+                print(f"Error parsing line: {line}")
+    return shas
+
+
+def get_file_info_by_sha(sha: str) -> Tuple[str, str]:
+    try:
+        with open(TARGET_FILE, 'r', encoding='utf-8') as f:
+            data = json.load(f)
+
+        for ext_name, versions in data["extensions"].items():
+            for version in versions:
+                if version["sha256Digest"] == sha:
+                    return version["filename"], version["downloadUrl"]
+
+        return None, None
+
+    except FileNotFoundError:
+        print(f"Error: {TARGET_FILE} not found")
+        raise
+    except json.JSONDecodeError:
+        print(f"Error: Invalid JSON format in {TARGET_FILE}")
+        raise
+
+
+def get_release_info(tag_name: str) -> Tuple[Optional[int], Optional[int]]:
+    try:
+        url = f"{base_url}/releases/tags/{tag_name}"
+
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+
+        data = response.json()
+
+        release_id = data.get("id")
+        release_body = data.get("body")
+        assets = data.get("assets", [])
+        asset_id = assets[0].get("id") if assets else None
+
+        return release_id, release_body, asset_id
+    except requests.RequestException as e:
+        print(f"API request failed: {e}")
+        return None, None, None
+    except (KeyError, IndexError) as e:
+        print(f"Failed to parse response data: {e}")
+        return None, None, None
+
+
+def update_release_body(release_id: int, commit_sha: str, old_body: str, sha: str, tag_name: str) -> bool:
+    try:
+        url = f"{base_url}/releases/{release_id}"
+        ref_url = f"{base_url}/git/refs/tags/{tag_name}"
```

Comment:
Since it is not possible to directly modify the relationship between the tag and commit in the release, I can only call the update ref API first to modify the corresponding relationship between the tag and commit.